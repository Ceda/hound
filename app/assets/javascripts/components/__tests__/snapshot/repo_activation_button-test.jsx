import React from 'react';
import renderer from 'react-test-renderer';

import RepoActivationButton from '../../repo_activation_button.js';

it('renders a button appropriately', () => {
  const repo = {
    id: 1,
    name: "Test repo",
    owner: {
      id: 1
    }
  }

  const onRepoClicked = jest.genMockFunction();

  const component = renderer.create(
    <RepoActivationButton
      repo={repo}
      onRepoClicked={onRepoClicked}
      isProcessingId={null}
    />

  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
