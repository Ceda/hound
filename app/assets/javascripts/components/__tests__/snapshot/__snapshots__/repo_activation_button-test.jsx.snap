exports[`test renders a button appropriately 1`] = `
<div
  className="repo-restricted">
  Only repo admins can activate
</div>
`;
